src/routes/safe/components/Balances/SendModal/screens/ReviewCollectible/index.tsx

```diff
@@ -57,12 +53,10 @@ const ReviewCollectible = ({ closeSnackbar, enqueueSnackbar, onClose, onPrev, tx
     let isCurrent = true
 
     const estimateGas = async () => {
-      const supportsSafeTransfer = await containsMethodByHash(tx.assetAddress, SAFE_TRANSFER_FROM_WITHOUT_DATA_HASH)
```

Comment:
can we leave this check? so if the method is not present in the code, we display a warning or error that the NFT can't be transferred using the `safeTransferFrom` (without data) method.

If not, we need to try/catch the later `await ERC721Token.at(tx.assetAddress)` call?

Reply:
ah, nvm. Thanks for the clarification @Agupane.

Reply:
it's done